test/source_dialogs_test.rb

```diff
@@ -7,7 +7,33 @@
 describe Yast::SourceDialogs do
   subject { Yast::SourceDialogs }
 
+  ZYPP_URL_MAPPINGS = [
```

Comment:
A mapping maps one value to another one, this is a plain list. I'd prefer a better name, like `ZYPP_VALID_URLS` or something like that.